### PR TITLE
Fixed the scrolling on the MHPGeneralScreen.

### PR DESCRIPTION
**Pull request by aikewoody**
# Description

https://www.notion.so/joinzoe/2021-08-17-Mental-health-playback-34d38dff55c74ea0aca3892ff0790e60?p=9287e6a43134499d8dce1912f714c152

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

[comment]: <> (Please attach screenshots showing the change, if relevant)

## Checklist (for submission)

- [ ] Analytics/tracking events have been added (if relevant)
- [ ] Added new screen(s) to `src/features/screens.ts` and new Redux slice(s) to `mocks/mockedInitialState.ts` (if relevant)

## Checklist (for reviewers)

- [ ] Checked against Figma designs (if relevant)
- [ ] Checked that data has been successfully saved to the backend (if relevant)

## Out of scope and potential follow-up

[comment]: <> (Are there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?)

### CODE REVIEW

File: src/features/mental-health-playback/components/Insights.tsx [hunk 1]
```diff
@@ -146,9 +141,8 @@ const styles = StyleSheet.create({
   },
   card: {
     alignSelf: 'center',
-    marginBottom: 'auto',
     marginHorizontal: 14,
-    marginTop: 'auto',
+    marginTop: 14,
```
Comment (bionara) [inline src/features/mental-health-playback/components/Insights.tsx:145]: should this come from Grid?

**Review by bionara — APPROVED** [review]
Code looks good!

File: src/features/mental-health-playback/components/Insights.tsx (hunk 1 above)
Comment (aikewoody) [reply src/features/mental-health-playback/components/Insights.tsx:145]: Yes and even worse it's not a multiple of 8 👎 

Somehow the 14 fits pixel perfect, besides that I want to release the feature asap so I left it like it is but these things should be cleaned with the coming design system refactor!

**Review by selvach — APPROVED** [review]
Tested on device. LGTM

**Pull request merged**